### PR TITLE
Adding properties to the request

### PR DESCRIPTION
**Pull request by nitinmotgi**

### CODE REVIEW

File: wrangler-service/src/main/java/co/cask/wrangler/service/directive/RequestDeserializer.java [hunk 1]
```diff
@@ -26,8 +26,10 @@
 import com.google.gson.JsonElement;
 import com.google.gson.JsonObject;
 import com.google.gson.JsonParseException;
+import com.google.gson.reflect.TypeToken;
```
Comment (CuriousVini) [inline wrangler-service/src/main/java/co/cask/wrangler/service/directive/RequestDeserializer.java]: nit: unused?

File: wrangler-service/src/main/java/co/cask/wrangler/service/directive/RequestDeserializer.java [hunk 2]
```diff
@@ -26,8 +26,10 @@
...
 
 import java.lang.reflect.Type;
+import java.util.Map;
```
Comment (CuriousVini) [inline wrangler-service/src/main/java/co/cask/wrangler/service/directive/RequestDeserializer.java]: nit: unused?

**Review by CuriousVini — APPROVED** [review]
Looks like there are unused imports. Rest LGTM 👍

**Comment by nitinmotgi** [comment]
@CuriousVini addressed comments.

**Comment by CuriousVini** [comment]
lgtm 👍

**Comment by nitinmotgi** [comment]
Thanks @CuriousVini, merging.

**Pull request merged**